tools/text_processing/split_file_to_collection/split_file_to_collection.xml

```diff
@@ -73,6 +91,8 @@
                 <option value="fastq">FASTQ</option>
                 <option value="tabular">Tabular</option>
                 <option value="fasta">FASTA</option>
+                <option value="txt">Text files</option>
+                <option value="generic">Genric</option>
```

Comment:
This should be generic I guess ... maybe a little bit more explanation ...?
Also you might want to add an example to the help?

Reply:
Sure. I extended the help text. Is this sufficient in your opinion?